### DISCUSSION TITLE
Cisco IMC (CIMC)

### DISCUSSION THREAD

**Discussion by YurichGu** [ruffle-rs/ruffle#3201] [Q&A]
Hi Colleagues,

Cisco used to use adobe flash at the management controller of servers (Cisco IMC).
Some servers were updated to use html5, but some don't have this update.

I tried to use ruffle browser extension to access to cisco cimc at servers with adobe flash version of cimc.
At the start it shows below warning 
![warn](https://user-images.githubusercontent.com/24411781/107643574-3b2ded00-6c6e-11eb-9ec7-80130bbef518.jpg)

I managed to click "Run Anyway", but It showed blue screen and doing nothing.
Moreover, some errors were generated. 
here is the file with those errors:
[console-export-2021-2-11_13-44-8.txt](https://github.com/ruffle-rs/ruffle/files/5965830/console-export-2021-2-11_13-44-8.txt)

Is there any chance to make ruffle working ?

Regards,
Yury

**Comment by Poopooracoocoo** [comment]
That means it was written ActionScript 3, the newer language for SWFs made after 2006. Ruffle started with AS1 and AS2 which runs in ActionScript Virtual Machine 1 (AVM1). There is some work on AS3 support but isn't visible. <3 The warning should probably explain this... tbh the new message is rather confusing and makes you just want to skip it. The previous one that mentioned AS3 was technical, but at least made you think before dismissing.

And woah, Cisco IMC looks _COMPLICATED_.

*but i could be wrong. i'm just tryna help >u<

**Reply by YurichGu** [reply]
thanks for the start ;-)
